contrib/storage-ipfs/README.zh.md

```diff
@@ -0,0 +1,184 @@
+# Drill Storage Plugin for IPFS
```

Comment:
Please keep only `README.md` in English, since it would be problematic to update it for other developers.

Reply:
@vvysotskyi 
Can they put a link to the Chinese version somewhere?

Reply:
But for this case, documentation outside the project also may be outdated.